gui/src/installer/view.rs

```diff
@@ -884,6 +885,131 @@ pub fn define_bitcoin<'a>(
+    exe_path: Option<&PathBuf>,
+    started: Option<&Result<std::process::Child, std::io::Error>>,
+) -> Element<'a, Message> {
+    let start_button = button::primary(None, "Start bitcoind").width(Length::Fixed(200.0));
+
+    let mut next_button = button::primary(None, "Next").width(Length::Fixed(200.0));
+    if let Some(Ok(_child)) = started {
+        // Next button is pressable even if stderr, e.g. in case bitcoind already running.
+        next_button = next_button.on_press(Message::Next);
+    };
+    layout(
+        progress,
+        "Start Bitcoin full node",
+        Column::new()
+            .push(if exe_path.is_some() {
+                Container::new(
+                    Row::new()
+                        .spacing(10)
+                        .align_items(Alignment::Center)
+                        .push(icon::circle_check_icon().style(color::GREEN))
+                        .push(text("bitcoind already installed").style(color::GREEN)),
+                )
+            } else {
+                Container::new(
+                    Row::new()
+                        .spacing(10)
+                        .align_items(Alignment::Center)
+                        .push(icon::circle_cross_icon().style(color::RED))
+                        .push(text("Cannot find bitcoind").style(color::RED)),
+                )
+            })
+            .push(
+                Row::new()
+                    .spacing(10)
+                    .push(Container::new(if exe_path.is_some() && started.is_none() {
+                        start_button.on_press(Message::StartInternalBitcoind(
+                            message::StartInternalBitcoindMsg::Start,
+                        ))
+                    } else {
+                        start_button
+                    }))
+                    .push(Row::new().spacing(10).push(next_button)),
+            )
+            .spacing(50)
+            .push_maybe(if started.is_some() {
+                started.map(|res| {
+                    if res.is_ok() {
+                        Container::new(
+                            Row::new()
+                                .spacing(10)
+                                .align_items(Alignment::Center)
+                                .push(icon::circle_check_icon().style(color::GREEN))
+                                .push(text("bitcoind started").style(color::GREEN)),
+                        )
+                    } else {
+                        Container::new(
+                            Row::new()
+                                .spacing(10)
+                                .align_items(Alignment::Center)
+                                .push(icon::circle_cross_icon().style(color::RED))
+                                .push(
+                                    text(res.as_ref().err().unwrap().to_string()).style(color::RED),
+                                ),
+                        )
+                    }
+                })
```

Comment:
It would be nice if the result of this appeared right after "bitcoind already installed", not after "Next".

![image](https://github.com/wizardsardine/liana/assets/22457751/a78d2905-07b9-4129-81b1-4816799a1107)

Reply:
I've updated this in https://github.com/wizardsardine/liana/commit/b152f9f7f147e13bfe43e3aeb4f714752d23d417.